config_src/coupled_driver/ocean_model_MOM.F90

```diff
@@ -559,6 +566,33 @@ subroutine add_berg_flux_to_shelf(G, fluxes, use_ice_shelf, density_ice, kv_ice)
                                    max(fluxes%mass_berg(i,j), fluxes%mass_berg(i,j+1)))
     enddo ; enddo
     call pass_vector(fluxes%frac_shelf_u, fluxes%frac_shelf_v, G%domain, TO_ALL, CGRID_NE)
+    
+    !Zero'ing out other fluxes under the tabular icebergs 
+    do j=jsd,jed ; do i=isd,ied
+        if (fluxes%frac_shelf_h(i,j) > 0.5) then  !Only applying for ice shelf covering most of cell
```

Comment:
Will this affect any other experiments? If we make this "0.5" a run-time parameter we could at least turn off the zero'ing.